### PR TITLE
Add Swish operator

### LINKED ISSUE #5853: Request for Swish Op
# Swish/SiLU
Do you have any plans to implement the Swish Op in ONNX?

### Describe the operator
Swish is a popular Activation fuction. Its mathematical definition could be found at https://en.wikipedia.org/wiki/Swish_function

TensorFLow has https://www.tensorflow.org/api_docs/python/tf/nn/silu
Keras has https://keras.io/api/layers/activations/ (also in https://www.tensorflow.org/api_docs/python/tf/keras/activations/swish)

Pytorch has https://pytorch.org/docs/stable/generated/torch.nn.SiLU.html 

### Can this operator be constructed using existing onnx operators?
Yes, it could be implemented as a combination of Mul and Sigmoid Ops:
 x * Sigmoid (beta * x)

### Is this operator used by any model currently? Which one?
Yes. Modern Yolo series like yolov5, yolov7, yolov8, yolop and EfficientNet all have such Swish Ops.  

Yolov5: https://github.com/ultralytics/yolov5/blob/master/models/tf.py#L224 

EfficientNet:
https://paperswithcode.com/method/efficientnet which has Swish in https://github.com/lukemelas/EfficientNet-PyTorch/blob/2eb7a7d264344ddf15d0a06ee99b0dca524c6a07/efficientnet_pytorch/model.py#L294 

### Are you willing to contribute it? (Y/N)
Possibly Yes.

### PR DESCRIPTION
**Pull request by isdanni**
### Description
These changes have been made to support the SiLU/Swish operator as a function op.

### Motivation and Context
Close #5853 

The SiLU function can be defined as $x * Sigmoid (alpha * x)$ where $alpha = 1.0$.

Swish ops is used in Yolo series and EfficientNet.

### Note

Discussion thread in the original PR: [#5964](https://github.com/onnx/onnx/pull/5964).  
A new PR is being opened because one comment in that thread [could not be resolved via force-push](https://github.com/onnx/onnx/pull/5964#issuecomment-3113936094).

### CODE REVIEW

**Comment by isdanni** [comment]
@justinchuby new PR here, closed the old one :)

**Review by justinchuby — APPROVED** [review]
Thank you!
